packages/cli/tests/unit/rpc/explain/collect-location-context.spec.ts

```diff
@@ -61,53 +58,73 @@ describe('collectLocationContext', () => {
         ]
       `);
 
-      expect(utils.exists).toHaveBeenCalledTimes(3);
-      expect(utils.exists).toHaveBeenCalledWith('/src/file1.js');
-      expect(utils.exists).toHaveBeenCalledWith('/lib/file1.js');
-      expect(utils.exists).toHaveBeenCalledWith('/src/file2.js');
+      expect(stat.mock.calls).toStrictEqual([
+        ['/src/file1.js'],
+        ['/lib/file1.js'],
+        ['/src/file2.js'],
+      ]);
       // note file in /other is skipped
     });
 
+    it('handles directory listings', async () => {
+      stat.mockResolvedValue({ isDirectory: () => true, isFile: () => false } as Stats);
+      jest.spyOn(fs, 'readdir').mockResolvedValue(['file1.js', 'file2.js'].map(mockDirent));
+
+      const result = await collectLocationContext(sourceDirectories, [Location.parse('/src:0')]);
+      expect(result).toEqual([
+        {
+          type: 'directory-listing',
+          content: 'file1.js\nfile2.js',
+          location: '.:0',
```

Comment:
What is the `:0` syntax and can that be dropped? I saw it in the logs and I didn't understand it :-)